### ISSUE TITLE
Handle and Bar Issue with Ionic

### ISSUE THREAD

**Issue by johannesschobel** [leongersen/noUiSlider#1076]
Dear @leongersen ,

first of all, i would like to thank you for this awesome library. I have used it in various projects and i really (!) really like it. Very easy to use and straight forward.

In a recent `ionic` project, i also tried to use `nouislider`, but stumbled upon some weird issues, i would like to discuss with you.

When i include the `10.0.0` version of `nouislider`, respective application looks as follows (initial state after booting):
![image](https://user-images.githubusercontent.com/9431350/83272505-829d4780-a1cb-11ea-8bb8-6ca644cae4fe.png)

However, if i use any other version HIGHER than 10.0.0, like, for example, `14.0.0` it fails. The respective application, then, looks as follows (initial state):
![image](https://user-images.githubusercontent.com/9431350/83272727-d0b24b00-a1cb-11ea-88fb-59bfa4c50d26.png)

Note the handle that is set to the **top** of the track, however, it shows the `50` value. If i try to move the handle, it looks as follows:
![nouislider-issue](https://user-images.githubusercontent.com/9431350/83273205-88dff380-a1cc-11ea-9857-1fff5b55dcc6.gif)

I also provided a repository to show the described behaviour:
https://github.com/johannesschobel/nouislider-issue
The repository contains information how to run respective application.

Am i missing something or using the library in a wrong way?
Thank you very much for your time and effort in solving this issue.

All the best

**Comment by leongersen** [comment]
Hi @johannesschobel, I haven't been at a proper computer for the weekend, so I haven't been able to check out your example yet. Sorry about that. 

Since you mention upgrading from noUiSlider 10.0.0, the main pointer I can give you is to revisit the slider CSS. Consider updating the stylesheet to the latest version. In noUiSlider 11, the handle positioning switched to using `transform` (to be able to benefit from hardware acceleration) over the previous top/bottom offsets. From your screenshots, it seems likely you are overriding either the `transform` or the `top` property for the `.noUi-origin` class.

**Comment by johannesschobel** [comment]
Dear @leongersen ,
thank you for your reply - i didn't expect to work during the weekend ;) no worries! :laughing: 

Thanks for pointing me into a direction, although it did not solve my issue properly. I searched, if respective styles where overwritten somewhere, however, i did not find anything. Keep in mind, that the minimal repository i provided is a blank `Ionic` application!

I did manage to solve the issue, however, by adding the following css rules:
```css
.noUi-connect {
    height: auto !important;
}

.noUi-origin {
    top: auto !important;
}
```

Does this make any sense to you?

All the best and thanks for your help!
